app/src/main/java/emu/skyline/input/onscreen/OnScreenItemDefinitions.kt

```diff
@@ -64,7 +64,7 @@ class JoystickButton(
     defaultRelativeRadiusToX,
     R.drawable.ic_button
 ) {
-    private val innerButton = CircularButton(onScreenControllerView, buttonId, config.relativeX, config.relativeY, defaultRelativeRadiusToX * 0.75f, R.drawable.ic_stick)
+    val innerButton = CircularButton(onScreenControllerView, buttonId, config.relativeX, config.relativeY, defaultRelativeRadiusToX * 0.75f, R.drawable.ic_stick)
```

Comment:
I don't see any reason why the private identifier was removed, but it may be that I'm blind.

Reply:
I'm using it in increase and decrease functions to change innerButton opacity too, i've changed it to internal to be more restrictive

Reply:
Yeah then I'm definitely blind, modify as you see fit.